torch_np/_dtypes.py

```diff
@@ -343,21 +343,40 @@ def __setstate__(self, value):
 # ### Defaults and dtype discovery
 
 
-def default_int_type():
-    return dtype(_dtypes_impl.default_int_dtype)
-
-
-def default_float_type():
-    return dtype(_dtypes_impl.default_float_dtype)
-
-
-def default_complex_type():
-    return dtype(_dtypes_impl.default_complex_dtype)
-
-
-def get_default_dtype_for(dtyp):
-    torch_dtype = dtype(dtyp).torch_dtype
-    return _dtypes_impl.get_default_type_for(torch_dtype)
+def set_default_dtype(fp_dtype="numpy", int_dtype="numpy"):
+    """Set the (global) defaults for fp and int dtypes.
```

Comment:
Actually, it'd be good to be more explicit at the top of the docs, rather than having people looking for the actual information.
```suggestion
    """Set the (global) defaults for fp, complex, and int dtypes.
    
    The complex dtype is inferred from the float (fp) dtype. It has 
    a width at least twice the width of the float dtype, 
    i.e., it's complex128 for float64 and complex64 for float32.
```